From 6926894b87e1a0e759ab1a3d7e28b879aa3c0030 Mon Sep 17 00:00:00 2001
From: jinwu <jinwu@codeaurora.org>
Date: Tue, 6 Feb 2018 13:29:50 +0800
Subject: [PATCH 26/67] Fix the issue of view don't reload when onPause

Just let the view go normally when menu of via as show.

Change-Id: I31f3e702b7f42cbc86a90eee4ea1510e721121c1
---
 src/com/android/gallery3d/ui/ActionModeHandler.java | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/com/android/gallery3d/ui/ActionModeHandler.java b/src/com/android/gallery3d/ui/ActionModeHandler.java
index 30489f9e2..0b367d7dc 100644
--- a/src/com/android/gallery3d/ui/ActionModeHandler.java
+++ b/src/com/android/gallery3d/ui/ActionModeHandler.java
@@ -197,7 +197,6 @@ public class ActionModeHandler implements Callback, PopupList.OnPopupItemClickLi
             } else if (action == R.id.action_share) {
                 String shareTitle = mActivity.getResources().
                         getString(R.string.share_dialogue_title);
-                mActivity.isTopMenuShow = true;
                 mActivity.startActivity(Intent.createChooser(
                         shareIntent, shareTitle));
                 return true;
-- 
2.17.1

